cdap-data-fabric/src/main/java/co/cask/cdap/data/runtime/DataSetsModules.java

```diff
@@ -23,16 +23,22 @@
 import co.cask.cdap.data2.dataset2.DatasetFramework;
 import co.cask.cdap.data2.dataset2.DefaultDatasetDefinitionRegistry;
 import co.cask.cdap.data2.dataset2.InMemoryDatasetFramework;
+import co.cask.cdap.data2.metadata.writer.BasicLineageWriter;
+import co.cask.cdap.data2.metadata.writer.LineageWriter;
+import co.cask.cdap.data2.metadata.writer.LineageWriterDatasetFramework;
 import com.google.inject.Module;
 import com.google.inject.PrivateModule;
 import com.google.inject.Scopes;
 import com.google.inject.assistedinject.FactoryModuleBuilder;
+import com.google.inject.name.Names;
 
 /**
  * DataSets framework bindings
  */
 public class DataSetsModules extends RuntimeModule {
 
+  public static final String BASIC_DATASET_FRAMEWORK = "basicDatasetFramework";
```

Comment:
Ok, maybe I missed it. Will the ``DatasetFramework`` named as ``basicDatasetFramework`` be used by anything other than the ``LineageDatasetFramework``?